vector/src/main/java/im/vector/app/features/rageshake/BugReporter.kt

```diff
@@ -259,13 +259,23 @@ class BugReporter @Inject constructor(
                         ReportType.SUGGESTION          -> "[Element] [Suggestion] $bugDescription"
                         ReportType.SPACE_BETA_FEEDBACK -> "[Element] [spaces-feedback] $bugDescription"
                         ReportType.AUTO_UISI_SENDER,
-                        ReportType.AUTO_UISI           -> "[AutoUISI] $bugDescription"
+                        ReportType.AUTO_UISI           -> bugDescription
                     }
 
                     // build the multi part request
                     val builder = BugReporterMultipartBody.Builder()
                             .addFormDataPart("text", text)
-                            .addFormDataPart("app", "riot-android")
+                            .apply {
+                                when (reportType) {
+                                    ReportType.AUTO_UISI_SENDER,
+                                    ReportType.AUTO_UISI           -> {
+                                        addFormDataPart("app", "element-auto-uisi")
+                                    }
+                                    else -> {
+                                        addFormDataPart("app", "riot-android")
+                                    }
+                                }
+                            }
```

Comment:
OK... You already expect me to say something about that, no? :)

Reply:
I mean having something like`.addFormDataPart("app", getRageshakeApp(reportType))` would be nicer

Reply:
you read my mind, or did I read yours?

Reply:
done